examples/README.md

```diff
@@ -13,3 +13,40 @@ that can be used as a starting point for new apps.
 If you think something is missing here or would like more examples of something
 then please
 [raise an issue](https://github.com/facultyai/dash-bootstrap-components/issues/new?template=feature.md).
+
+## How to run the examples
+
+### Python
+
+Install the Python requirements from the file using
+
+```
+pip install -r requirements.txt
+```
+
+Then run the file using 
+
+```
+python app.py
+```
+
+### Julia
+
+Install the Julia requirements from the file within Julia using:
+
+```julia
+pkg> activate .
+pkg> instantiate
+```
+
+Once the requirements are installed, run the file from Julia using:
+
+```julia
+julia> include("julia_file_name.jl")
+```
+
+Or you can run from any terminal using:
```

Comment:
I think maybe any -> the since it needs to be run from the directory containing the `Project.toml` etc?